packages-exp/auth-exp/src/core/user/user_impl.test.ts

```diff
@@ -116,9 +128,22 @@ describe('core/user/user_impl', () => {
   });
 
   describe('#delete', () => {
```

Comment:
can you test the side effect on stsTokenManager?

Reply:
Done